### PR TITLE
Added font family and autoCapitalize props

### PR DESCRIPTION
**Pull request by dcgoss**
Allows for customization of the font. Value defaults to 'System' font.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -206,7 +212,7 @@ export default class Search extends Component {
                 style={[
                   styles.input,
                   {
-                    color: textColor, marginLeft: hideBack ? 30 : 0,
+                    color: textColor, fontFamily: this.props.fontFamily, marginLeft: hideBack ? 30 : 0,
```
Comment (daveols) [inline index.js]: Please import this prop like the others have been at the top of the render function

File: index.js [hunk 2]
```diff
@@ -218,6 +224,7 @@ export default class Search extends Component {
                 underlineColorAndroid='transparent'
                 returnKeyType='search'
                 autoCorrect={this.props.autoCorrect}
+                autoCapitalize={this.props.autoCapitalize}
```
Comment (daveols) [inline index.js]: Same as with this one (also change autoCorrect while you're there! Just noticed that)

File: index.js [hunk 3]
```diff
@@ -60,6 +63,7 @@ export default class Search extends Component {
     clearOnHide: true,
     focusOnLayout: true,
     autoCorrect: true,
+    autoCapitalize: 'none'
```
Comment (daveols) [inline index.js]: It should default to what React Native defaults to, as to not break current implementations. I believe this is 'sentences'

File: index.js [hunk 4]
```diff
@@ -49,6 +51,7 @@ export default class Search extends Component {
     backgroundColor: 'white',
     iconColor: 'gray',
     textColor: 'gray',
+    fontFamily: 'System',
```
Comment (daveols) [inline index.js]: You sure this works as a default? Can't see it in the React Native docs.

**Comment by dcgoss** [comment]
Sounds good. Will make the changes this evening.

File: index.js (hunk 4 above)
Comment (dcgoss) [inline index.js]: https://github.com/facebook/react-native/issues/1611

https://github.com/facebook/react-native/blob/c1458e01816690be224acb4576aa48f0e505a446/React/Views/RCTFont.mm#L246

**Comment by dcgoss** [comment]
I believe that addresses the critiques from above.

File: index.js (hunk 4 above)
Comment (daveols) [inline index.js]: Cool, thanks

**Comment by daveols** [comment]
@dcgoss waiting on another PR then I'll release this for you. Stay tuned.

**Pull request merged**